src/Homepage/index.js

```diff
@@ -72,7 +71,6 @@ class Homepage extends React.Component {
   async fetchResults() {
     let selectedLocation;
```

Comment:
You were right about confusing variable names. should we make it more descriptive, something like
`locationComingFromStoreSubmitCall`